swap_numbers.py

```diff
@@ -0,0 +1,13 @@
+#code to swap two number in a fun way
+
+a = 5
```

Comment:
Could you please change the variable a to 42?

Reply:
Sure, let's do that.